tests/backup/backup_helper.go

```diff
@@ -117,6 +117,7 @@ const (
 	cloudCredConfigMap                        = "cloud-config"
 	volumeSnapshotClassEnv                    = "VOLUME_SNAPSHOT_CLASS"
 	rancherActiveCluster                      = "local"
+	rancherProjectDescription                 = "new project"
```

Comment:
Is it ok to keep this constant?

Reply:
There is not much importance of the description and it can be same for all the project.
Also, if we need to have any specific description in any test case, that is always an option